src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -4803,64 +4789,62 @@ def vmss_run_command_create(client,
                             output_blob_uri=None,
                             error_blob_uri=None,
                             no_wait=False):
-    run_command = {}
-    run_command['location'] = location
+    from .aaz.latest.vmss.run_command import Create
+    args = {}
+    args['location'] = location
+    args['resource_group'] = resource_group_name
+    args['run_command_name'] = run_command_name
+    args['instance_id'] = instance_id
+    args['vmss_name'] = vmss_name
+    args['no_wait'] = no_wait
     if tags is not None:
-        run_command['tags'] = tags
-    source = {}
+        args['tags'] = tags
     if script is not None:
-        source['script'] = script
+        args['script'] = script
     if script_uri is not None:
-        source['script_uri'] = script_uri
+        args['script_uri'] = script_uri
```

Comment:
Could you please double-check the parameter structure? I think `script_uri` might belong under the `source` field.

Reply:
`source` is flattened for input args